skins/cat17/package.json

```diff
@@ -59,26 +74,33 @@
     "tap-min": "^1.1.0",
     "tape": "^4.9.1",
     "tape-catch": "^1.0.6",
+    "uglify-js": "^2.5.0",
     "vinyl-buffer": "^1.0.1",
     "vinyl-source-stream": "^2.0.0",
+    "vueify": "^9.0.0",
     "watchify": "^3.11.0"
   },
   "dependencies": {
     "bootstrap-sass": "^3.3.7",
-    "bootstrap-select": "^1.13.1",
+    "bootstrap-select": "^1.13.2",
     "jcf": "^1.2.3",
     "jquery": "^3.3.1",
     "object-assign": "^4.0.1",
+    "polyfill": "^0.1.0",
     "promise": "^7.3.1",
     "redux": "^3.7.2",
     "redux-promise": "^0.5.3",
     "underscore": "^1.9.1",
-    "underscore.path": "^0.1.4"
+    "underscore.path": "^0.1.4",
+    "vue": "^2.5.17"
   },
   "browserify": {
+    "browser": {
+      "vue": "vue/dist/vue.common.js"
+    },
     "transform": [
       [
-        "polyify",
+        "vueify",
```

Comment:
Ok, so in this setup we've dropped `polify`, which is ok because that was only for IE <10 support. This PR should then remove it from the dev packages in package.json and also drop the configuration object that follows this line.